### PR TITLE
[ONNX] Update the documentation for the onnx exporter with a known issue

### PR DESCRIPTION
**Pull request by xadupre**
Extends the documentation to fix the conversion to onnx of model [BasicUNet](https://docs.monai.io/en/stable/networks.html#basicunet).

### CODE REVIEW

File: docs/source/onnx_dynamo.rst [hunk 1]
```diff
@@ -149,8 +150,41 @@ The main advantages are:
...
+        f = torch.jit.isinstance
+        torch.jit.isinstance = isinstance
+
+        try:
+            yield
+        finally:
+            torch.jit.isinstance = f
+
+
+        with bypass_export_jit_isinstance():
```
Comment (justinchuby) [inline docs/source/onnx_dynamo.rst:183]: Should we just do it for users?

Comment (xadupre) [reply docs/source/onnx_dynamo.rst:183]: I let you decide.

Comment (justinchuby) [reply docs/source/onnx_dynamo.rst:183]: I think we should do this for the users as there's no downside to it (as far as I can tell?) I can take this contextmanager to the capture_strategies file

**Comment by justinchuby** [comment]
Created https://github.com/pytorch/pytorch/pull/137592

**Pull request closed without merging**